ui/appui-react/src/appui-react/syncui/SyncUiEventDispatcher.ts

```diff
@@ -148,6 +150,10 @@ export class SyncUiEventDispatcher {
     SyncUiEventDispatcher._uiEventDispatcher.dispatchSyncUiEvent(SyncUiEventId.ViewStateChanged);
   }
 
+  private static _dispatchFeatureOverridesChange() {
```

Comment:
As the source line was not tested, please add similar `//istanbul ignore next` so the coverage remains